conf/test_full.config

```diff
@@ -40,5 +41,5 @@ params {
     phase          = false
 
     // Impute tools
-    tools = "glimpse1"
+    tools = "glimpse2"
```

Comment:
Why this change? I am confused as the reason behind this.

Reply:
I mean, we are going to launch a completely different AWS mega test from the one before.

Reply:
The aim is to reduce the computational cost of the megatest as much as possible.
The `glimpse2` software as less steps (no variant calling) and should also be more computationally efficient.
I'll add it to the changelog